packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,10 +7,65 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [37.0.0]
+
+### Added
+
+- **BREAKING:** Add elements to the `AssetsContractController` class: ([#4397](https://github.com/MetaMask/core/pull/4397))
```

Comment:
Nit: The same goes for this change (sorry for being unclear with the last comment). Even though we are "adding" options here, it's not a new feature. "Added" is for new features/capabilities.

Reply:
We do currently have "Adding an optional argument to an existing function or method" as an example of an "addition" changelog entry in our `reviewing-release-prs` doc.

> https://github.com/MetaMask/core/blob/main/docs/reviewing-release-prs.md#additions

Also, we're categorizing function parameter removals under the "Removed" heading instead of "Changed", implying that what's being removed there is a feature.

In general, I think it makes sense to see function parameter additions as expanding the consumer-facing surface of the API. In the case of this entry, we could maybe go further and see it as implementing a new feature as well: add support for the messenger pattern by adding a new capacity to the class -- the ability to accept messengers.

But if we want to be more strict about how we define "features" in this context, or prevent breaking changes from being categorized as additions, I think that makes sense too, so long as we're being consistent. What are your thoughts on this? Should we update the `reviewing-release-prs` doc with new guidelines about additions?

Reply:
Note that in that doc, it's talking about adding _optional_ arguments, but this entry is breaking because we've added a required argument.

Though you do raise a good point, the doc doesn't exactly match my description of the category, maybe there is room for improvement there.  The category definitions are in the Keep A Changelog docs, linked at the top of each changelog.

Reply:
Makes sense that a new feature would rarely be a required/breaking change and vice versa. I'm good with recategorizing this.

Fixed here! https://github.com/MetaMask/core/pull/4548/commits/d7d79852af6f48b352f59eeb5f172f3908255e87